packages/worker-activity-page/src/controllers/activity.tsx

```diff
@@ -0,0 +1,40 @@
+import {ActivityPage} from "../components/activity-page" 
+import {ErrorPage} from "../components/error-page"
+
+export default class Activity {
+
+    private _existToken: string;
+
+    constructor(existToken: string) {
+        this._existToken = existToken;
+    }
+
+
+    async getActivity() {
+        const query = [
+            'attributes=events,commits,tweets',
+            'limit=31'
+        ]
+
+        try {
+            const activities = await fetch(`https://exist.io/api/1/users/$self/attributes/?${query.join('&')}`, {
+                method: "GET",
+                headers: {
+                    "Authorization": `Token ${this._existToken}`,
+                    "Content-Type": "application/json"
+                }
+            }).then(res => res.json())
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/alexwilson/frontend/pullRequest?prid=7165000)
```suggestion
            }).then((res) => res.json())
```